package.json

```diff
@@ -46,10 +46,10 @@
     "@react-navigation/native-stack": "^7.2.0",
     "@reduxjs/toolkit": "^2.5.1",
     "i18next": "^24.2.2",
-    "react": "18.3.1",
+    "react": "19.0.0",
     "react-dom": "18.3.1",
```

Comment:
`react-dom` also seems to have a matching version for react-native. Could you please try upgrading it to 19.0.0 and see if it works? Thanks!